### PR TITLE
feat(minor): Various outstanding fixes

### PR DESCRIPTION
**Pull request by hassila**
## Description

Fixes for the following (as separate commits)
https://github.com/ordo-one/package-benchmark/issues/202
https://github.com/ordo-one/package-benchmark/issues/201
https://github.com/ordo-one/package-benchmark/issues/195
https://github.com/ordo-one/package-benchmark/issues/194
https://github.com/ordo-one/package-benchmark/issues/191
https://github.com/ordo-one/package-benchmark/issues/198

## How Has This Been Tested?

Test suite, manual testing with https://github.com/wadetregaskis/Swift-Benchmarks

## Minimal checklist:

- [x] I have performed a self-review of my own code 
- [x] I have added `DocC` code-level documentation for any public interfaces exported by the package
- [ ] I have added unit and/or integration tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Plugins/BenchmarkCommandPlugin/BenchmarkPlugin+Help.swift [hunk 1]
```diff
@@ -52,7 +52,7 @@ let help =
                           "retainReleaseDelta", "custom"]
     --path <path>           The path where exported data is stored, default is the current directory (".").
     --quiet                 Specifies that output should be suppressed (useful for if you just want to check return code)
-    --scale                 Specifies that some of the text output should be scaled using the scalingFactor (denoted by '*' in output)
+    --scale                 Show the metrics without scaling for the inner loop (scalingFactor will not be applied to output)
```
Comment (blindspotbounty) [inline Plugins/BenchmarkCommandPlugin/BenchmarkPlugin+Help.swift:55]: Is it correct that now it is `no-scale`? :)

**Review by blindspotbounty — APPROVED** [review]

File: Plugins/BenchmarkCommandPlugin/BenchmarkPlugin+Help.swift (hunk 1 above)
Comment (hassila) [reply Plugins/BenchmarkCommandPlugin/BenchmarkPlugin+Help.swift:55]: Yeah, conceptually you are right. Perhaps I'll add no-scale and make scale a no-op.

**Review by dimlio — APPROVED** [review]

File: Plugins/BenchmarkTool/BenchmarkTool+ReadP90AbsoluteThresholds.swift [hunk 2]
```diff
@@ -43,7 +43,7 @@ extension BenchmarkTool {
             path = cwdPath
         }
 
-        var p90Thresholds: [String: BenchmarkThresholds.AbsoluteThreshold]?
+        var p90Thresholds: [String: BenchmarkThresholds.AbsoluteThreshold]? = nil
```
[the remark this answers is not in the document]
Comment (hassila) [reply Plugins/BenchmarkTool/BenchmarkTool+ReadP90AbsoluteThresholds.swift:46]: You are right, I probably prefer to be explicit though for consistency (just saw that [Slava Pestov pitched this to be removed years ago](https://forums.swift.org/t/but-it-is-an-uninitialized-optional/68882/5), but unfortunately it was never done, too late now... )

File: Sources/Benchmark/OperatingSystemStats/OperatingSystemStatsProducer+Linux.swift [hunk 3]
```diff
@@ -197,6 +199,11 @@
 
                     sampleSemaphore.signal()
```
[the remark this answers is not in the document]
Comment (hassila) [reply Sources/Benchmark/OperatingSystemStats/OperatingSystemStatsProducer+Linux.swift:200]: Yeah, bad copy/paste from the Darwin implementation, thanks.

**Pull request merged**